src/components/contributions-table.jsx

```diff
@@ -139,13 +149,22 @@ class ContributionsTable extends React.Component {
 
     return (
       <div>
-        <input value={this.state.filterField} onChange={updateFilter} type="text" />
+        <input className="filter" value={this.state.filterField} onChange={updateFilter} type="text" placeholder="Type to search" />
         <table className="contributors">
           <thead className="contributors__thead">
             <tr>
-              <td className="contributors__name"><button onClick={sortToggle('name')}>^</button>Name</td>
-              <td className="contributors__amount"><button onClick={sortToggle('amount')}>^</button>Amount</td>
-              <td className="contributors__date contributors__col--s1"><button onClick={sortToggle('date')}>^</button>Date</td>
+              <td className={`contributors__name${isActive('name')}`} onClick={sortToggle('name')}>
+                <span className="arrow-container" />
+                <input type="button" className="sort-column" value="Name" />
+              </td>
+              <td className={`contributors__amount${isActive('amount')}`} onClick={sortToggle('amount')}>
```

Comment:
You probably just want to move the `onClick` back to the button to avoid the jsx-a11y issues and then wrap the arrow within the button.